docs/source/_static/dpf_entry.html

```diff
@@ -0,0 +1,3407 @@
+
+<h3 class="h3-main">DPF's helpers to access mechanical's data</h3>
+<p class="p-main">Diverse Mechanical�s and DPF's data can be accessed: </p>
+<ul class="ul-main">
+    <li  id="des_PackageACTDoc">this documentation can be generated via:</li>
+    <li id="des_PackageACTDataSources">the result files of Mechanical�s analysis: to collect the DataSources (DPF�s entity containing result file paths) the user can write:</li>
+    <li id="des_PackageACTMeshScoping">a mesh selection in the interface: to collect a mesh Scoping (DPF�s entity representing a list of ids of nodes or elements), once a geometry selection is picked out in the interface, it can be accessed via:</li>
+</ul>
+
+<h2 class="h2-main">Python Result</h2>
+<p class="p-main">The Python Result object allows to evaluate a workflow and to visualize its outputs in Mechanical (contours/graph/mesh...).</p>
+<h3 class="h3-main">Enable the Python Result</h3>
+<p class="p-main">To enable its use, the feature flags �Beta Options� (tab Tools/Appearance) and �Create Python Results� (tab Tools/Mechanical) must be checked in Workbench interface. From Mechanical, a Python Result object can be inserted under the Solution object using contextual menu or using Model ribbon menu.</p>
+<h3 class="h3-main">Evaluate a Python Result</h3>
+<p class="p-main">Once the python result object is added (see previous part) and written (a template script to calculate the Total Deformation result at the last step is provided by default) and the analysis is solved, an evaluation is needed .Any changes to the script�s contents will make the state of the Python Result object go �Undefined�. Users must then �Connect� the callback back to the event before evaluating by right clicking on the python result object and selecting "Connect". The user can finally click on "Solve" or "Evaluate Result".</p>
+<h3 class="h3-main">Write a Python Result</h3>
+<p class="p-main" id="des_WritePythonResult">The Workflow is the global entity that will be uses. Built by chaining operators, it will evaluate the data defined by the used operators. It needs input information, and it will compute the requested output information. This "output information" is then used by mechanical to plot contours/tables/charts and meshes on the application. To chose the information that will be used by mechanical, the user must expose results out of his workflow. To do so, the workflow's API give the ability to "SetOutputContour(my_operator)" for contour (the colors) plotted on the geometry, to "SetOutputWarpField(my_operator)" to deforme the contours (with 3D displacements) and to "SetOutputMesh(my_operator)" to plot the results on a different mesh than the current mesh in Mechanical. If "SetOutputContour(my_operator)" publishes several results over time in a fields container, a chart and a table will plot the min and max over time. Here is an example using the 3 output types, other examples can be found under "APIs"/"Worflow Examples":</p>
```

Comment:
The Workflow is the global entity that is used. Built by chaining operators, it evaluates the data defined by the operators used in the chain. Supply it with input information, and it will compute the requested output information. This output information is then used by Mechanical to plot contours, tables, charts, and meshes in the application. To select the information that will be used by Mechanical, expose the results of your workflow. To do so, use the following worklfow API output types:  

- `SetOutputContour(my_operator)` for contour (the colors) plotted on the geometry
- `SetOutputWarpField(my_operator)` to deform the contours (with 3D displacements)
-` SetOutputMesh(my_operator)` to plot the results on a different mesh than the current mesh in Mechanical. 

If `SetOutputContour(my_operator)` publishes several results over time in a fields container, a chart and a table will plot the min and max over time. Here is an example using the three output types, other examples can be found under **APIs/Worflow Examples**

Reply:
updated in TFS